### ISSUE TITLE
What to do after flushing firmware? OpenLase and Jack integration with sd rgb 500.

### ISSUE THREAD

**Issue by PanKuskus** [maximus64/sd_rgb500_cfw#3]
I followed your instructions and flushed Yours firmware to my sd rgb 500. Can you give some information on what to do next. I supose it should be recognized as audio device? And then I could transmit data from openlase to it with use of jack server? Right now i have device which is recognized as Laser Controller but nothing shows in jack server. Do I need some drivers so it will be recognized properly. Sorry for so much questions but I would apriciate some more info about whole project because it is amazing and sorry if I missed something obvious. Thanks.

**Comment by PanKuskus** [comment]
Ok I found Jack_bridge.c file in host folder (yesterday at 3 am I was probably to sleepy) but it would be nice if you gave some more info in Readme.  Can't wait to test it when I come back home. Cheers.

**Comment by maximus64** [comment]
You are correct. The jack_bridge is the piece that take samples from Jack server and send it to over USB. It not using USB audio interface, just raw isochronous transfers. Good luck and let me know if you get stuck :)
